paper/src/main/java/com/denizenscript/denizen/paper/properties/EntityFriction.java

```diff
@@ -0,0 +1,85 @@
+package com.denizenscript.denizen.paper.properties;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import io.papermc.paper.entity.Frictional;
+import net.kyori.adventure.util.TriState;
+
+public class EntityFriction implements Property {
+
+    public static boolean describes(ObjectTag entity) {
+        return entity instanceof EntityTag
+                && ((EntityTag) entity).getBukkitEntity() instanceof Frictional;

```

Comment:
`instanceof EntityTag entityTag && entityTag.getBukkitEntity()...` (or change the param name to `object` and use `entity` as the variable name here)

Reply:
what

Reply:
New Java feature that we can use now that Denizen is Java 16+, `pattern matching in instanceof` if you'd like to goole it